packages/core/src/components/dialog/dialog.tsx

```diff
@@ -17,6 +17,7 @@
 import classNames from "classnames";
 import * as React from "react";
 import { polyfill } from "react-lifecycles-compat";
+import { uniq } from "lodash-es";
```

Comment:
We don't depend on lodash in core. I don't really want to add that dependency now, so we'll need to come up with our own "unique ID" utility function. it can live in packages/core/src/common/utils/jsUtils.ts

Also, did you mean to use `uniqueId` instead of `uniq` here?